attest/core/src/error.rs

```diff
@@ -239,9 +239,14 @@ pub enum QuoteError {
     InvalidUtf8,
 }
 
-impl From<ConvertError> for QuoteError {
-    fn from(src: ConvertError) -> Self {
-        QuoteError::Encoding(src.into())
+impl From<base64::DecodeError> for QuoteError {
+    fn from(src: base64::DecodeError) -> Self {
+        use base64::DecodeError::*;
+        let kind = match src {
+            InvalidLength => EncodingError::InvalidInputLength,
+            InvalidByte(_, _)|InvalidLastSymbol(_, _) => EncodingError::InvalidInput,
+        };
+        QuoteError::Encoding(kind)
```

Comment:
I notice EncodingError already implements `From<base64::DecodeError>` so we can further simplify this
```suggestion
        QuoteError::Encoding(src.into())
```